mlos_bench/mlos_bench/tests/tunable_groups_fixtures.py

```diff
@@ -62,6 +62,7 @@
                 "type": "int",
                 "default": 2000000,
                 "range": [0, 1000000000],
+                "quantization": 1000000,
```

Comment:
Probably need tests to check this now, right?

Reply:
absolutely. I'll add a ton more unit tests to this PR shortly